### ISSUE TITLE
error Unresolved function or method initAccessibility()

### ISSUE THREAD

**Issue by AndreyAndreev1380** [ahmadnassri/node-nightwatch-accessibility#23]
when setting up tests initAccessibility() is highlighted with error `Unresolved function or method initAccessibility()` and if running tests error is:"✖ Failed [fail]: (aXe failed to execute)
    at processTicksAndRejections (internal/process/task_queues.js:93:5) 

 Error while running .setTimeoutsAsyncScript() protocol action: unknown command: unknown command: timeouts/async_script

 Error while running .executeScriptAsync() protocol action: unknown command: unknown command: execute_async"
my test is:
`module.exports
    = {
   '@tags': ['accessibility'],

    'Test Create Benefits detail page':  function(browser){
        browser
            .url(siteUrl)
            .pause(1000)
            .setValue('input[name="name"]', name)
            .setValue('input[name="pass"]', pass)
            .click('input[id="edit-submit"]')
            .url(siteUrl+'/node/add/page')
            .waitForElementVisible('.page-title', 6000)
            .assert.title('Create Benefits detail page | VA CMS')
            .pause(5000)
            .initAccessibility()
            .assert.accessibility(axeOptions, contextOptions)
            .end(function(err, res){
                console.log(res, err);
            });
    },

};`
and nightwatch.json is 
`{
    "src_folders" : ["./tests/nightwatch/features"],
    "output_folder" : "./tests/nightwatch/reports",
    "custom_commands_path": ["./node_modules/nightwatch-accessibility/commands"],
    "custom_assertions_path": ["./node_modules/nightwatch-accessibility/assertions"],
    "page_objects_path" : "./tests/nightwatch/pageobjects",
    "globals_path" : "globalsModule.js",

    "selenium" : {
        "start_process" : false,
        "server_path" : "",
        "log_path" : "",
        "port" : 9515,
        "cli_args" : {
            "webdriver.chrome.driver" : "./node_modules/chromedriver/lib/chromedriver",
            "webdriver.gecko.driver" : "",
            "webdriver.edge.driver" : ""
        }
    },

    "test_settings" : {
        "default" : {
            "launch_url" : "http://va-gov-cms.lndo.site/",
            "selenium_port"  : 9515,
            "selenium_host"  : "localhost",
            "default_path_prefix" : "",
            "silent": true,
            "screenshots" : {
                "enabled" : false,
                "path" : ""
            }
        },
        "desiredCapabilities": {
            "browserName": "chrome",
            "chromeOptions" : {
                "args" : ["headless", "--no-sandbox"]
            },
            "acceptSslCerts": true
        },
        "chrome" : {
            "desiredCapabilities": {
                "browserName": "chrome"
            }
        }
    }
}
`
"nightwatch": "^1.2.2",
    "nightwatch-accessibility": "^1.8.0",
nodeJs v12.10.0

**Comment by Snowballers** [comment]
We're running into exactly the same problem with a bit more recent version of Nightwatch and Chromedriver. Did you ever figure it out?

**Comment by ahmadnassri** [comment]
the latest release (v1.10.0) should have this addressed
